certified-connectors/Tikit/apiDefinition.swagger.json

```diff
@@ -1741,1435 +1779,2903 @@
-      "type": "object",
-      "required": [
-        "Email"
-      ],
-      "properties": {
-        "Email": {
-          "description": "Change Assignee Email",
-          "x-ms-summary": "Assignee Email",
-          "type": "string",
-          "title": "Assignee Email"
-        }
+        "produces": [
+          "application/json"
+        ],
+        "summary": "GetTicketLifecyclePhase",
+        "operationId": "GetTicketLifecyclePhase",
+        "description": "Get a list of all Lifecycle Phases associated to the ticket",
+        "x-ms-visibility": "internal",
+        "parameters": [
+          {
+            "name": "Id",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "x-ms-summary": "Tikit Id",
+            "description": "Tikit Id"
+          }
+        ]
       }
     },
-    "ServiceDesk.Core.Actions.UpdateTicket.TicketDTO": {
-      "type": "object",
-      "required": [
-        "Id"
-      ],
-      "properties": {
-        "Id": {
-          "description": "Enter Ticket ID",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Enter Ticket ID",
-          "title": "Ticket ID"
-        },
-        "Title": {
-          "description": "Change Request Information",
-          "type": "string",
-          "x-ms-summary": "Change Request Information",
-          "title": "Request"
-        },
-        "Requester": {
-          "$ref": "#/definitions/ServiceDesk.Core.Actions.UpdateTicket.CdmUserDTO"
-        },
-        "StatusId": {
-          "description": "Change Status",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Status",
-          "x-ms-dynamic-values": {
-            "operationId": "StatusList_Get",
-            "value-path": "Id",
-            "value-collection": "value",
-            "value-title": "Value"
-          },
-          "title": "Status"
+    "/Ticket({Id})/GetPowerAutomateTasks": {
+      "get": {
+        "responses": {
+          "200": {
+            "description": "Response from Get Tasks",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "value": {
+                  "type": "array",
+                  "items": {
+                    "$ref": "#/definitions/ServiceDesk.Core.Models.PowerAutomateTask"
+                  }
+                }
+              }
+            }
+          }
         },
-        "CategoryId": {
-          "description": "Change Category",
-          "type": "integer",
-          "format": "int32",
-          "x-ms-summary": "Category",
-          "x-ms-dynamic-values": {
-            "operationId": "CategoryId_Get",
-            "value-collection": "value",
-            "value-path": "Id",
-            "value-title": "Value"
+        "summary": "Get Power Automate Tasks",
+        "operationId": "GetPowerAutomateTasks",
+        "description": "Get PowerAutomate Tasks",

```

Comment:
The keyword Power Automate is not allowed, can you please remove
The value 'summary' contains at least one of the restricted words: 'power apps, power automate, logic apps'. ValuePath: /paths/~1Ticket({Id})~1GetPowerAutomateTasks/get/summary